docker-compose/docker-compose.yml

```diff
@@ -206,10 +204,6 @@ services:
      expose:
        - 46070
      privileged: true
-     environment:
```

Comment:
you've removed these lines...are you sure they are not needed? did you double check with @syc17 ?

Reply:
Sorry, I don't have the context to the change.  My components do not call identification at the moment.  I am still waiting for instructions from UPC regarding the CAU call to validate IDKey.

Reply:
When I read your message I originally thought I had made a mistake, but now that I see the [original file](https://github.com/mF2C/mF2C/blob/master/docker-compose/docker-compose.yml) I think I did not... lines 210-212 are in the resource-categorization section, so I guess I dont need to double check this with @syc17 (do i need to?) but with @ssgUPC and I did it.

Reply:
Don't worry :) and yes, I already confirmed it with @ssgUPC